component/azstorage/utils.go

```diff
@@ -131,8 +131,8 @@ func getSDKLogOptions() policy.LogOptions {
 	} else {
 		// TODO:: track2 : check which headers and query params should not be redacted
 		return policy.LogOptions{
-			AllowedHeaders:     []string{"x-ms-version"},
-			AllowedQueryParams: []string{},
+			AllowedHeaders:     allowedHeaders,
```

Comment:
by default does it log something or everything is redacted ?

Reply:
By default, these headers are logged, https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/runtime/policy_logging.go#L38-L64

And these query params are logged, https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/runtime/policy_logging.go#L81-L83

Reply:
The list we provided here are the additional headers and query params that we want to be logged and not redacted.